client/my-sites/sidebar-unified/style.scss

```diff
@@ -119,6 +126,68 @@ $font-size: rem( 14px );
 	.current-site .site,
 	.current-site .all-sites,
 	.current-site__switch-sites {
-		border-color: transparent;
+		border-color: #333333;
```

Comment:
Is there a var equivalent?

Reply:
Very good finding! https://github.com/Automattic/wp-calypso/pull/45829/commits/0fa8f23eec05dc64c82a404581bfe9dfa815b3da